tests/stdlib/trstgen.nim

```diff
@@ -1326,6 +1326,53 @@ Test1
           output)
     check("""<th align="left">-d</th><td align="left">option</td>""" in
           output)
+
+  test "Roles: subscript prefix/postfix":
+    let outputPrefix  = "See :subscript:`some text`.".toHtml
+    let outputPostfix = "See `some text`:subscript:.".toHtml
+    check(outputPrefix == outputPostfix)
+    check(outputPrefix == "See <sub>some text</sub>.")
+
+  test "Roles: correct parsing from beginning of line":
+    let output1 = """:superscript:`3`\ He is an isotope of helium.""".toHtml
+    check(output1 == "<sup>3</sup>He is an isotope of helium.")
+    let output2 = """:sup:`3`\ He is an isotope of helium.""".toHtml
+    let output3 = """`3`:sup:\ He is an isotope of helium.""".toHtml
+    let output4 = """`3`:superscript:\ He is an isotope of helium.""".toHtml
+    check(output2 == output1)
```

Comment:
nit for future PR: prefer `check a == b` to `check(a == b)` unless parsing requires it
(ditto with doAssert)

Reply:
ok, thanks.